src/pages/Settings/RecoveryCodeConfirm.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+	<kv-lightbox
+		:visible="visible"
+		title="2-step verification is on"
+		@lightbox-closed="closeLightbox"
+		prevent-close="true"
+	>
+		<p>
+			Copy this recovery code and keep it somewhere safe. You'll
+			need it if you want to log in without your device.
+		</p>
+		<div class="recovery-code">
+			{{ mfaRecoveryCode }}
+		</div>
+		<div>
+			<kv-button
+				class="secondary smallest expanded copy-code"
+				@click.native.prevent="copyToClipboard(mfaRecoveryCode)"
+			>
+				Copy code
+			</kv-button>
+		</div>
+		<div>
+			<kv-checkbox
+				id="codeVerify"
+				class="verified-checkbox"
+				radio-value="I have safely recorded this code"
+				v-model="verified"
+			>
+				I have safely recorded this code
+			</kv-checkbox>
+		</div>
+		<div>
+			<kv-button
+				class="expanded smaller show-for-small-only"
+				:disabled="isDisabled"
+				@click.native.prevent="closeLightbox"
+				aria-label="Close small"
+			>
+				Continue
+			</kv-button>
+
+			<kv-button
+				class="expanded smaller show-for-medium"
+				:disabled="isDisabled"
+				@click.native.prevent="closeLightbox"
+				aria-label="Close large"
```

Comment:
Same, no need for an aria label because we have readable text inside the button. The label would be useful if you had an icon inside but no text.

Reply:
At some point during development, storybook was throwing an accessibility error asking for these fields. Thanks for noticing and explaining why they are not needed. They've been removed in the latest commit.